### DISCUSSION TITLE
Update on Auth Success GPIO Time not working

### DISCUSSION THREAD

**Discussion by Miker2022** [rednblkx/HomeKey-ESP32#83] [Q&A]
Hello,

thanks a lot for this great project!
I would like to use the Auth Success GPIO PIN to control my lock. Unfortunately I cannot update the "Auth Success GPIO Time" wie the WEB GUI. It alwaysd sticks at 1000ms.

Any Idea? Is thata bug?

Michael

**Comment by Miker2022** [comment]
Just realized the using the Auth Success GPIO  for controlling access is probably a bad idea since Auth success is triggered on every succesful transaction even with arbitrary Mifare-Cards - not only succesful Homekey transactions.

Could we add config option to restrict GPIO-actions on succesful Homekey transactions only?

**Reply by rednblkx** [reply]
If you assign a pin on “GPIO Actions” it will be pulled LOW or HIGH(per your option) on every Homekey successful event and only Homekey and will also set the state of the Homekey entity.

**Comment by rednblkx** [comment]
Hi!

Yes, there seems to be a bug with setting the timeout. Will be fixed.
